CHANGELOG.md

```diff
@@ -4,7 +4,9 @@
 - Setting the REVISION environment variable is now optional ([#429](https://github.com/Shopify/kubernetes-deploy/pull/429))
 - Defaults KUBECONFIG to `~/.kube/config` ([#429](https://github.com/Shopify/kubernetes-deploy/pull/429))
 - Uses `TASK_ID` environment variable as the `deployment_id` when rendering resource templates for better [Shipit](https://github.com/Shopify/shipit) integration. ([#430](https://github.com/Shopify/kubernetes-deploy/pull/430))
-- Arguments to `--bindings` will now be deep merged. ([#419](https://github.com/Shopify/kubernetes-deploy/pull/419))
+- Arguments to `--bindings` will now be deep merged. ([#430](https://github.com/Shopify/kubernetes-deploy/pull/430))
```

Comment:
This is re-instating an error. Needs rebasing.

https://github.com/Shopify/kubernetes-deploy/commit/d0343e7571471fc8d3c1b4d05296698fd70ac06f